raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpanToStructuredTraceAvroGroupAggregator.java

```diff
@@ -3,15 +3,29 @@
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
 import java.util.List;
+
+import com.typesafe.config.Config;
 import org.apache.avro.generic.IndexedRecord;
 import org.apache.flink.api.common.functions.AggregateFunction;
 import org.hypertrace.core.datamodel.RawSpan;
 import org.hypertrace.core.datamodel.StructuredTrace;
+import org.hypertrace.core.datamodel.TimestampRecord;
+import org.hypertrace.core.datamodel.shared.DataflowMetric;
 import org.hypertrace.core.datamodel.shared.trace.StructuredTraceBuilder;
 
 public class RawSpanToStructuredTraceAvroGroupAggregator implements
     AggregateFunction<RawSpan, List<RawSpan>, StructuredTrace> {
 
+  private double timestampSamplingPercent = -1;
+  private final String TIMESTAMP_SAMPLING_PERCENT = "timestamp.sampling.percent";
```

Comment:
How about calling it with more specific name, like `dataflow.metriccollection.sampling.percent`?

Reply:
done